internal/logutils/logutils.go

```diff
@@ -0,0 +1,18 @@
+package logutils
+
+import (
+	"io"
+	"os"
+
+	"github.com/gittuf/gittuf/internal/logutils/slogger"
+	"github.com/mattn/go-colorable"
```

Comment:
Overall, I think I lean towards dropping this. I've not been convinced in the past on colouring log output and I lean towards not introducing the dependency. Is this something others feel strongly about? cc @wlynch as well.

Reply:
I have dropped the colors, as everybody agrees on it.